package-lock.json

```diff
@@ -1,8 +1,9201 @@
 {
   "name": "@workablehr/orka",
   "version": "3.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "@workablehr/orka",
```

Comment:
@klesgidis I am guessing you installed packages with an older npm in the past. I have verion 8.5.0 installed you?